index.js

```diff
@@ -22,6 +23,7 @@ class Organizations extends Component {
     super(props, context);
     this.connectedApp = props.stripes.connect(Main);
     this.connectedContactsContainer = props.stripes.connect(ContactsContainer);
+    this.connectedInterfaceContainer = props.stripes.connect(InterfaceRouterContainer);
```

Comment:
Could you please stick to the existing pattern? There is no `Router` for contacts

Reply:
fixed